.github/workflows/java.yml

```diff
@@ -25,48 +25,46 @@ on:
             - .github/workflows/install-valkey/action.yml
             - .github/json_matrices/build-matrix.json
     workflow_dispatch:
+        inputs:
+            full-matrix:
+                description: "Run the full matrix"
+                required: false
+                default: "false"
+
+    schedule:
+        - cron: "0 0 * * *"
```

Comment:
Can you shift hours please to later time? Midnight in UTC occurs at 16:00 in our TZ. CI getting stuck on PRs due to occupied runners.
Keep in mind, `valkey` repo has number of crons on midnight too.